experimental/web_dashboard/lib/src/pages/home.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020, the Flutter project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+
+import '../api/api.dart';
+import 'item_details.dart';
+
+class HomePage extends StatelessWidget {
+  Widget build(BuildContext context) {
+    var api = Provider.of<DashboardApi>(context);
+
+    return Scaffold(
+      body: StreamProvider<List<Item>>(
```

Comment:
Could a StreamBuilder be used here instead of the StreamProvider/Consumer combo?

Reply:
I originally did that, but this is going to change later anyway, so I think we should leave this as-is for now.

Reply:
Fair enough.